src/containers/Permissions/index.jsx

```diff
@@ -93,6 +95,11 @@ class Permissions extends React.Component {
 		if (form.get('isEditMode') !== prevForm.get('isEditMode')) {
 			this.props.formPermissionKeys();
 		}
+
+		if (permissionsKeys !== prevPermissionsKeys) {
+			const keyWeight = await this.props.checkKeyWeightWarning(networkName, accountId);
```

Comment:
in case of changing wif from other sources we cannot saw warning message until rooted to permission page, i think we should return check to subscribe function to handle this extra case